src/Traits/DetectsChanges.php

```diff
@@ -25,6 +25,10 @@ protected static function bootDetectsChanges()
 
     public function attributesToBeLogged(): array
     {
+        if (isset(static::$logFillable)) {
+            return $this->fillable;
```

Comment:
Shouldn't we merge the fillable attributes with the ones specified in `logAttributes`?

Reply:
Yes, right 👍  should work that way